src/lambda/mod.rs

```diff
@@ -19,8 +16,5 @@ pub use evaluation_context::EvaluationContext;
 pub use expression::*;
 pub use graphql_operation_context::GraphQLOperationContext;
 pub use io::*;
-pub use list::*;
 pub use logic::*;
```

Comment:
Can drop logic and move the `is_truthy` function into `helpers::value`

Reply:
actually we can drop the entire logic.rs as `is_truthy` is only used in tests.